### PR TITLE
Fix apache tika download reference

### LINKED ISSUE #167: Fix Apache Tika URL
Currently, the URL to download the Apache Tika in our container image to run the spider changes with some annoying frequency. Every time a version is released the old one is deleted and our URL is not valid anymore. We need to find a way to make it more stable. In the current version, the URL is maintained by Unicamp. 

On top of my head I think we can look for a URL which keeps more than one version at time. Thus, the binary will be available for more time.

### PR DESCRIPTION
**Pull request by hugoleodev**
Dockerfile references apache tika from unicamp FTP host
 This commit changes apache tika download from unicamp FTP to
 root Apaches download URLs

Closes #167

### CODE REVIEW

**Comment by hugoleodev** [comment]
@jvanz Travis checks for black violations. The violations are in https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/settings.py and https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/spiders/rj_rio_de_janeiro.py

I think that problem should be fixed in another issue.

**Comment by jvanz** [comment]
> @jvanz Travis checks for black violations. The violations are in https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/settings.py and https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/spiders/rj_rio_de_janeiro.py
> 
> I think that problem should be fixed in another issue.

Yes, there are some files with problems. Thanks for the report. If you want, you can send the fixes via PR.  We can open an issue for that as well. ;)

**Comment by hugoleodev** [comment]
> > @jvanz Travis checks for black violations. The violations are in https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/settings.py and https://github.com/okfn-brasil/diario-oficial/blob/master/processing/data_collection/gazette/spiders/rj_rio_de_janeiro.py
> > I think that problem should be fixed in another issue.
> 
> Yes, there are some files with problems. Thanks for the report. If you want, you can send the fixes via PR. We can open an issue for that as well. ;)

I will fix it.

**Comment by jvanz** [comment]
By the way, thanks for your first contribution! 

giphy2

**Comment by hugoleodev** [comment]
Roses are Red! PRs are all green!

File: .gitignore [hunk 1]
```diff
@@ -109,3 +109,6 @@ ENV/
 
 # scrapy deltafetch middleware directory
 data/deltafetch
+
+# vscode
+.vscode
```
Comment (jaswdr) [inline .gitignore:114]: can you remove this? is not related with the PR

**Comment by jvanz** [comment]
> @hugoleodev can you squash your commits?

@hugoleodev, I would say squash the commits fixing the black violations. You can keep the change in the dockerfile in a separate commit.

File: .gitignore (hunk 1 above)
Comment (hugoleodev) [reply .gitignore]: Yes! I have changed!

**Comment by hugoleodev** [comment]
Done @jvanz @jaswdr,  I squashed the commits.

**Comment by jaswdr** [comment]
> Done @jvanz @jaswdr, I squashed the commits.

:shipit:

**Comment by jvanz** [comment]
\o/

**Pull request merged**